pkg/karmadactl/promote/promote.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 The Karmada Authors.
+Copyright 2024 The Karmada Authors.
```

Comment:
The test file needs to be modified.

Reply:
Thank you for pointing that out! It looks like I missed it. 🙏